G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Commands/OutlookGetRootFoldersCommand.cs

```diff
@@ -13,12 +13,12 @@
 
 namespace G1ANT.Addon.MSOffice
 {
-    [Command(Name = "outlook.getrootfolders", Tooltip = "This command is used to return a list of all Outlook root folders.")]
+    [Command(Name = "outlook.getrootfolders", Tooltip = "This command is used to return a list of all Outlook root folders")]
     public class OutlookGetRootFoldersCommand : Command
     {
         public class Arguments : CommandArguments
         {
-            [Argument(Required = true, Tooltip = "Name of a variable where the command's result will be stored. The variable will be of [list](G1ANT.Language/G1ANT.Language/Structures/ListStructure.md) structure containing elements of [outlookfolder](G1ANT.Addon/G1ANT.Addon.MSOffice/G1ANT.Addon.MSOffice/Structures/OutlookFolderStructure.md) structure")]
+            [Argument(Required = true, Tooltip = "Name of a variable where the command's result will be stored. The variable will be of list structure containing elements of outlookfolder structure")]
```

Comment:
Name of a variable where the command's result will be stored (a list of elements of outlookfolder structure)